src/app/layout.tsx

```diff
@@ -0,0 +1,49 @@
+import type { Metadata } from "next";
+import { Space_Mono, Work_Sans } from "next/font/google";
+import "./globals.css";
+import Image from "next/image";
+import Link from "next/link";
+import Button from "@/components/button/button";
+
+const space_mono = Space_Mono({
+    subsets: ["latin"],
+    weight: ["400", "700"],
+    display: "swap",
+    variable: "--font-space-mono",
+});
+
+const work_sans = Work_Sans({
+    subsets: ["latin"],
+    weight: ["400", "600"],
+    display: "swap",
+    variable: "--font-work-sans",
+});
+
+export const metadata: Metadata = {
+    title: "NFT Marketplace",
+    description: "NFT Marketplace",
+};
+
+export default function RootLayout({ children }: { children: React.ReactNode }) {
+    return (
+        <html lang="en">
+            <body className={`${space_mono.variable} ${work_sans.variable}`}>
```

Comment:
You can add `flex flex-col min-h-screen` to `<body>` and you can add `grow` to children. It will fix this problem when the page is scrollable.

https://github.com/JackHamish/marketplace_front/assets/37477697/0fc119c9-c45b-460f-a66c-7a0e54816c06